content/04.use-cases-b.md

```diff
@@ -75,13 +73,12 @@ Additional GitHub Actions can be integrated with Manubot, such as ones creating
 
 <!--*contributors to this section:* Eric R. Scott-->
 
-Peer review is the standard process for assessing and judging whether research done in ecology and evolution should be published in a scientific journal.
+Peer review is the standard process for assessing whether research done in ecology and evolution should be published in a scientific journal.
 GitHub provides an open and transparent platform that can be used for either directly providing feedback on research products or addressing changes recommended by reviewers.
-GitHub Issues can be used to organize and discuss reviewer suggestions and to assign them to co-authors (_e.g._, <https://github.com/SORTEE-Github-Hackathon/manuscript/issues?q=label%3A%22Reviewer+Comment%22+>).
-When reviewer comments are posted as separate issues, authors can comment on the issues to discuss possible changes and assign themselves to indicate which comments they intend to handle.
-Co-authors can then integrate their edits and responses to reviewers using pull requests.
+GitHub Issues can be used to organize and discuss reviewer suggestions and to assign them to co-authors (_e.g._, <https://github.com/SORTEE-Github-Hackathon/manuscript/issues?q=label%3A%22Reviewer+Comment%22>).
+When reviewer comments are posted as separate issues, authors can comment on the issues to discuss possible changes and assign those that are going to address the issue.
+Co-authors can then integrate their edits and responses to reviewers using pull requests, which can be directly linked to the issues they address.
 
 GitHub can also assist reviewers during the peer review process.
 If the code associated with a manuscript is made available at the time of submission (_e.g._, as a link to a GitHub repository within the Data Availability Statement), peer reviewers may be able to offer more comprehensive suggestions on the code and written materials, potentially recognizing errors before publication.
-Certain journals or software development communities require submitted work or research code to be hosted on GitHub and their review processes make use of GitHub Issues (_e.g._, rOpenSci (<https://ropensci.org/software-review/>), Journal of Open Source Software (<https://joss.readthedocs.io/en/latest/submitting.html>)).
 rOpenSci's efforts have resulted in many well-used R packages for ecology research including `rfishbase` [@doi:10.1111/j.1095-8649.2012.03464.x] and `taxize` [@doi:10.12688/F1000RESEARCH.2-191.V2].
```

Comment:
Either remove this sentence also or make it fit better with the previous sentence.